docs/domainmetadata.rst

```diff
@@ -172,6 +172,14 @@ This metadata can also be set using the
 and ``set-publish-cds``. For an example for an :rfc:`7344` key rollover,
 see the :doc:`guides/kskrollcdnskey`.
 
+.. _metadata-slave-renotify:
+
+SLAVE-RENOTIFY
+--------------
```

Comment:
Please add:

```
.. versionadded:: 4.3.0
```

Reply:
done, and fixed doc to mention the new logic